docs/basic-docs/atomicdex/atomicdex-api.md

```diff
@@ -688,7 +688,7 @@ This command must be executed at the initiation of each MM2 instance. Also, Atom
 
 ::: tip
 
-Electrum mode is available for utxo-based coins only; this includes Bitcoin and Bitcoin-based forks. Electrum mode is not available for ETH/ERC20.
+Electrum mode is available for utxo-based and QRC20-based coins only; this includes Bitcoin and Bitcoin-based forks. Electrum mode is not available for ETH/ERC20.
```

Comment:
I think it's better to call `QRC20-based` as `QRC20 tokens`, because QRC20 are based on QTUM.

Reply:
Done